src/components/workspace/Editor.tsx

```diff
@@ -130,34 +134,30 @@ class Editor extends React.PureComponent<IEditorProps, {}> {
             // SharedbAceMultipleCursors
           ]
         );
+        if (this.props.sharedbAceIsInviting) {
```

Comment:
Consider using guard clauses for this whole file for your collab editor changes!

Have a read here: https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a / https://refactoring.com/catalog/replaceNestedConditionalWithGuardClauses.html 

e.g. line 121

```
if (this.props.editorSessionId ===  ' ') return;

//rest of logic
```